prow/cmd/webhook-server/helpers.go

```diff
@@ -246,37 +254,183 @@ func createValidatingWebhookConfig(ctx context.Context, caPem string, client ctr
 	if err != nil && strings.Contains(err.Error(), configAlreadyExistsError) {
 		logrus.Info("ValidatingWebhookConfiguration already exists, proceeding to patch")
 		if err := patchValidatingWebhookConfig(ctx, caPem, client); err != nil {
-			return fmt.Errorf("failed to patch validation webhook config: %w", err)
+			return fmt.Errorf("failed to patch validating webhook config: %w", err)
 		}
 	} else if err != nil {
-		return fmt.Errorf("failed to create validation webhook config: %w", err)
+		return fmt.Errorf("failed to create validating webhook config: %w", err)
 	}
 
 	return nil
 }
 
 func patchValidatingWebhookConfig(ctx context.Context, caPem string, client ctrlruntimeclient.Client) error {
 	key := types.NamespacedName{
-		Namespace: "default",
-		Name:      "prow-job-validating-webhook-config.prow.k8s.io",
+		Namespace: defaultNamespace,
+		Name:      prowJobValidatingWebhookName,
 	}
 
 	patchOptions := &ctrlruntimeclient.PatchOptions{
 		FieldManager: "webhook-server",
 	}
 	var validatingWebhookConfig admregistration.ValidatingWebhookConfiguration
 	if err := client.Get(ctx, key, &validatingWebhookConfig); err != nil {
-		return fmt.Errorf("failed to get validation webhook config: %w", err)
+		return fmt.Errorf("failed to get validating webhook config: %w", err)
 	}
 	oldValidatingWebhook := validatingWebhookConfig.DeepCopy()
 	validatingWebhookConfig.Webhooks[0].ClientConfig.CABundle = []byte(caPem)
 	if err := client.Patch(ctx, &validatingWebhookConfig, ctrlruntimeclient.MergeFrom(oldValidatingWebhook), patchOptions); err != nil {
-		return fmt.Errorf("failed to patch validation webhook config: %w", err)
+		return fmt.Errorf("failed to patch validating webhook config: %w", err)
+	}
+	return nil
+}
+
+func createMutatingWebhookConfig(ctx context.Context, caPem string, client ctrlruntimeclient.Client) error {
+	operations := []admregistration.OperationType{"CREATE"}
+	scope := admregistration.ScopeType("*")
+	path := "/mutate"
```

Comment:
this needs to match the path defined in http handler from this service, make a const for it and use it in both places